src/app/content/components/CallToActionPopup/styles.ts

```diff
@@ -0,0 +1,124 @@
+import styled from 'styled-components';
+import { Times } from 'styled-icons/fa-solid/Times/Times';
+import Button from '../../../components/Button';
+import { textRegularStyle } from '../../../components/Typography';
+import { textStyle } from '../../../components/Typography/base';
+import theme from '../../../theme';
+
+// tslint:disable-next-line: variable-name
+export const CTAWrapper = styled.div`
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  z-index: 99;
+  width: 100%;
+
+  @media screen and (max-width: 1200px) {
+    display: none;
+  }
+`;
+
+// tslint:disable-next-line: variable-name
+export const CTAContent = styled.div`
+  position: absolute;
+  z-index: 3;
```

Comment:
this you can leave inline because it is part of the fixed subtree, not the global z-index stack

Reply:
I've changed this to `1` but it is needed if I want to keep components in current order because if I would like this to be on top then I would have to move `<CTABackground>` at the beginning